clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -393,6 +396,47 @@ class GCEUtil {
     return ((tags ?: []) + securityGroupTags).unique()
   }
 
+  static Boolean shouldUpdateUrlMap(UrlMap urlMap, UpsertGoogleLoadBalancerDescription description) {
```

Comment:
This method is complicated enough that it would benefit from thorough unit tests.